src/lib/tradingView/utils.ts

```diff
@@ -8,21 +8,37 @@ import { Themes } from '@/styles/themes';
 
 import { AppTheme, type AppColorMode } from '@/state/configs';
 
+import { testFlags } from '../testFlags';
+
 export const mapCandle = ({
   startedAt,
   open,
   close,
   high,
   low,
   baseTokenVolume,
-}: Candle): TradingViewBar => ({
-  time: new Date(startedAt).getTime(),
-  low: parseFloat(low),
-  high: parseFloat(high),
-  open: parseFloat(open),
-  close: parseFloat(close),
-  volume: Math.ceil(Number(baseTokenVolume)),
-});
+  trades,
+  orderbookMidPriceOpen,
+  orderbookMidPriceClose,
+}: Candle): TradingViewBar => {
+  const isOhlcEnabled = testFlags.ohlc;
+  const hasNoTrades = trades === 0;
+
+  return {
+    time: new Date(startedAt).getTime(),
+    low: parseFloat(low),
+    high: parseFloat(high),
```

Comment:
Should we change low and high as well?

`low - Min (midPriceOpen, midPriceClose)`
`high - Max (midPriceOpen, midPriceClose)`

Reply:
ah yes ty - good catch should def do that!